apps/api/src/app/subscribers/usecases/handle-chat-oauth/handle-chat-oauth.command.ts

```diff
@@ -0,0 +1,50 @@
+import { BaseCommand } from '@novu/application-generic';
+import {
+  IsEnum,
+  IsMongoId,
+  IsString,
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+} from 'class-validator';
+import { ChatProviderIdEnum } from '@novu/shared';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export function IsNotEmpty(validationOptions?: ValidationOptions) {
```

Comment:
What's the difference of this implementation with the `class-validator` `IsNotEmpty`?